var/spack/repos/builtin/packages/netpbm/package.py

```diff
@@ -170,6 +170,15 @@ def edit(self, spec, prefix):
 
     def build(self, spec, prefix):
         make()
+        if self.run_tests:
+            # Don't run the default command 'make check' for test
+            self.build_time_test_callbacks = []
+
+    @run_after('build')
+    @on_package_attributes(run_tests=True)
+    def test(self):
```

Comment:
This method is reserved for stand-alone tests.  Please give it a different name.

See https://spack.readthedocs.io/en/latest/packaging_guide.html#build-time-tests for more information on install-time tests (`build`, `install`).

Reply:
Docs on stand-one tests are at https://spack.readthedocs.io/en/latest/packaging_guide.html#stand-alone-or-smoke-tests.

Reply:
OK. I've changed the name. Please review it again. Thank you.